CHANGELOG.md

```diff
@@ -12,6 +12,12 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## 7.1.1
+
+### Fixed
+
+  * Enhancement of `7.0.5`: more use cases covered (to avoid repeatedly fetching a new token).
```

Comment:
```suggestion
  * Enhancement of `7.0.5`: more use cases covered (to avoid repeatedly fetching a new token).
  * When using remote config, the full local `idp-authentication` is now injected (some fields were missing).
```